support-frontend/assets/components/subscriptionCheckouts/subscriptionSubmitButtons.jsx

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+import React from 'react';
+
+import { type PaymentMethod } from 'pages/digital-subscription-checkout/digitalSubscriptionCheckoutReducer';
+import { type IsoCurrency } from 'helpers/internationalisation/currency';
+import { hiddenIf } from 'helpers/utilities';
+import { type SetupPayPalRequestType } from 'helpers/paymentIntegrations/payPalRecurringCheckout';
+import type { BillingPeriod } from 'helpers/billingPeriods';
+import { PayPalExpressButton } from 'components/paypalExpressButton/PayPalExpressButton';
+import Button from 'components/button/button';
+import { type Option } from 'helpers/types/option';
+
+// ----- Types ----- //
+
+type PropTypes = {|
+  paymentMethod: Option<PaymentMethod>,
+  currencyId: IsoCurrency,
+  csrf: CsrfState,
+  setupRecurringPayPalPayment: SetupPayPalRequestType,
+  payPalHasLoaded: boolean,
+  isTestUser: boolean,
+  onPaymentAuthorised: Function,
+  amount: number,
+  billingPeriod: BillingPeriod,
+  validateForm: Function,
+  formIsValid: Function,
+|};
+
+
+// ----- Render ----- //
+
+
+function SubscriptionSubmitButtons(props: PropTypes) {
+  return (
+    <div>
+      <div
+        id="component-paypal-button-checkout"
+        className={hiddenIf(props.paymentMethod !== 'PayPal', 'component-paypal-button-checkout')}
```

Comment:
can this be a ternary instead of using css?

```jsx
<div>
  {props.paymentMethod === 'PayPal' ? <PayPalExpressButton  ... /> :  < Button ... />}
</div>
```

Reply:
No because the PayPal loading is expensive and involves state changes so we only want to do it once. I've added a comment to explain it.